44/vipinreyo/marvel.py

```diff
@@ -0,0 +1,86 @@
+from collections import Counter, namedtuple
+import csv
+
+DATA = 'marvel-wikia-data.csv'
+
+Character = namedtuple('Character', 'pid name sid align sex appearances year')
+
+
+def convert_csv_to_dict(data=DATA):
+    '''write a function to parse marvel-wikia-data.csv, see
+       https://docs.python.org/3.7/library/csv.html#csv.DictReader
+       should return a list of OrderedDicts or a list of Character
+       namedtuples (see Character namedtuple above')'''
+    with open(data) as csv_file:
+        return [row for row in csv.DictReader(csv_file)]
+
+
+data = list(convert_csv_to_dict())
+
+
+def most_popular_characters(n=5):
+    '''get the most popular character by number of appearances
+       accept an argument of n (int) of most popular characters
+       to return (leave default of 5)'''
+    return [character['name'].split('(')[0].strip() for character in sorted(data, key=lambda x: int(x['APPEARANCES'])
```

Comment:
can this be written in more lines? `if x['APPEARANCES'] else 0` -> `if x.get('APPEARANCES', 0)` ?